net-im/psi/psi-9999.ebuild

```diff
@@ -36,23 +36,22 @@ RDEPEND="
 	dev-qt/qtgui:5
 	dev-qt/qtmultimedia:5
 	dev-qt/qtnetwork:5
+	dev-qt/qtsql:5[sqlite]
```

Comment:
fyi, some packages (notably those using private headers, such as akonadi) depending on qtsql[sqlite] were broken even on patch releases, which caused us to auto-bump the qtsql subslot with ${PN} full version. that's just a heads-up, and it does not mean you have to add `:=` here. but if you encounter the odd bug report...

Reply:
Ok, thanks